tests/core18/basic/task.yaml

```diff
@@ -0,0 +1,17 @@
+summary: Check basic core18 ssystem functionality
+
+execute: |
+    echo "Check that the system snaps are there"
+    snap list | MATCH core18
+    snap list | MATCH snapd
+    if snap list | grep '^core +'; then
+        echo "The old core snap is installed but should not"
+        exit 1
+    fi
+
+    echo "Ensure that the system is fully seeded"
+    snap changes | MATCH "Done.*Initialize system state"
+
+    echo "Check that the minimal test-snapd-busybox-static works"
+    snap install test-snapd-busybox-static
+    test-snapd-busybox-static.busybox-static echo hello | MATCH hello
```

Comment:
Question, why static? It should work in a dynamically linked world as well, right?

Reply:
I switched that now to "test-snapd-tools-core18" (which we can put under a track to hide the name). The reason for the static one was just that it was in the store with a minimal base.